compiler/syntax/src/res_token.ml

```diff
@@ -102,8 +102,8 @@ type t =
 let precedence = function
   | HashEqual | ColonEqual -> 1
   | Lor -> 2
-  | Land -> 3
-  | Caret -> 4
+  | Caret -> 3
+  | Land -> 4
```

Comment:
The table in https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Operator_precedence#table shows the order as

![Bildschirmfoto 2025-04-13 um 07 32 17](https://github.com/user-attachments/assets/b6dd078a-3bdc-486e-9be3-95fcef70f382)

though.

So the original Lor, Land, Caret order seems correct?

Reply:
You right, I had confused token names :disappointed: There is just no `Token.Bar` for binary expr yet.